### PR TITLE
added homepage as per issue #2

### LINKED ISSUE #2: Application Home Page

### PR DESCRIPTION
**Pull request by amadywr**
#2

- Created components folder
- Created button component inside components folders
- Created homepage component inside components folders
- Added button components inside the homepage component
- Updated the app.css file

### CODE REVIEW

File: frontend/src/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
```
Comment (RheshavMaharaj) [inline frontend/src/components/Button.tsx]: Unused import

File: frontend/src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      <a href={href} className={className}>
+        {text}
+      </a>
+    </div>
+  )
+}
+
+Button.defaultProps = {
+  text: 'Button',
+}
```
Comment (RheshavMaharaj) [inline frontend/src/components/Button.tsx]: Please use an interface to set up props, using default props will not allow intellisense to detect invalid props. Additionally, there are more props in this component that have not been listed and as a result can cause errors if they are not defined and the app is expecting them

File: frontend/src/components/Homepage.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import logo from '../imgs/logo1.png'
+import Button from './Button'
+
+const homepage = () => {
+  return (
+    <section className="section-hero">
+      <div className="container">
+        <div className="hero">
+          <header>
+            <a href="#">
```
Comment (RheshavMaharaj) [inline frontend/src/components/Homepage.tsx]: Add accessibility tag

**Review by RheshavMaharaj — CHANGES_REQUESTED** [review]

**Comment by amadywr** [comment]
removed button component as it was unnecessary and added accessibility to logo a tag

**Review by RheshavMaharaj — APPROVED** [review]
Looks good to me :D

**Pull request merged**